Sources/MapboxMaps/Foundation/Camera/CameraAnimationsManager.swift

```diff
@@ -57,43 +57,16 @@ public class CameraAnimationsManager {
 
     // MARK: Setting a new camera
 
-    /// Transition the map's viewport to a new camera.
+    /// Sets the map's viewport to a new camera synchronously. Any active animations will be canceled.
     /// - Parameters:
     ///   - targetCamera: The target camera to transition to.
-    ///   - animated: Set to `true` if transition should be animated. `false` by default.
-    ///   - duration: Controls the duration of the animation transition. Must be greater than zero if `animated` is true.
-    ///   - completion: The completion block to be called after an animated transition. Only called if `animated` is true.
-    public func setCamera(to targetCamera: CameraOptions,
-                          animated: Bool = false,
-                          duration: TimeInterval = 0,
-                          completion: ((UIViewAnimatingPosition) -> Void)? = nil) {
+    public func setCamera(to targetCamera: CameraOptions) {
```

Comment:
Later: I do wonder if we're going to need a completion block here anyway, to know when those camera settings have taken effect.